src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1885,11 +1885,16 @@ static void processEntity(ContextRegistrationResponse* crr, const EntityIdVector
   en.id   = getStringFieldF(entity, REG_ENTITY_ID);
   en.type = entity.hasField(REG_ENTITY_TYPE) ? getStringFieldF(entity, REG_ENTITY_TYPE) : "";
 
-  /* isPattern = true is not allowed in registrations so it is not in the
-   * document retrieved with the query; however we will set it to be formally correct
-   * with NGSI spec
+  /*
```

Comment:
Probably all the comment can be removed now.

Reply:
Yeah, I thought that too when I saw it in the PR

Reply:
Fixed in 3dc4bcf83f00f7937ce79cacd25417575f5beaed